package.json

```diff
@@ -65,12 +67,14 @@
     "whatwg-fetch": "^1.0.0"
   },
   "devDependencies": {
+    "css-loader": "^0.23.1",
```

Comment:
will we have issues with this like we had with the svg loader? if you are not sure, move it up to prod deps, and we can revisit later.